internal/broker/bindings/bindings_manager.go

```diff
@@ -137,3 +139,36 @@ func (c *ServiceAccountBindingsManager) Create(ctx context.Context, instance *in
 
 	return string(kubeconfigContent), expiresAt, nil
 }
+
+func (c *ServiceAccountBindingsManager) Delete(ctx context.Context, instance *internal.Instance, bindingID string) error {
+	clientset, err := c.clientProvider.K8sClientSetForRuntimeID(instance.RuntimeID)
+
+	if err != nil {
+		return fmt.Errorf("while creating a runtime client for binding creation: %v", err)
+	}
+
+	serviceBindingName := fmt.Sprintf("kyma-binding-%s", bindingID)
```

Comment:
We could extract it as separate function - single point of truth - like with Runtime CR name, or Kyma CR name.
But this is nice to have, not must.